Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -518,25 +517,49 @@ - (void)loadVungleNativeAdForParameters:(id<MAAdapterResponseParameters>)paramet
     return clickableViews;
 }
 
-- (BannerSize)adSizeFromAdFormat:(MAAdFormat *)adFormat
+- (VungleAdSize *)adSizeFromAdFormat:(MAAdFormat *)adFormat
+                          parameters:(id<MAAdapterParameters>)parameters
 {
-    if ( adFormat == MAAdFormat.banner )
+    BOOL isAdaptiveBanner = parameters.localExtraParameters[@"adaptive_banner"];
```

Comment:
Android checks if banner format first then check if is adaptive size.

Reply:
Yeah. I noticed that. Is there any particular reason for that?

Reply:
@HuFangshuai 
Updated to be similar to Android side.